.travis.yml

```diff
@@ -16,7 +16,7 @@
 language: php
 
 php:
-  - 7.1
+  - "7.1"
```

Comment:
are you sure about this? i have never seen it with quotes

Reply:
My IDE complains about non strings in YAML, what can go wrong? The versions in the matrix are doing their job.

Reply:
right, this is a yaml file :man_facepalming: 
agreed. i know some people who have choice words to say about yaml :-D 
its often a bit sloppy, i remember symfony having changed their parser to become more strict at some point because it did not complain about totally stupid things...
lets keep the quotes.